### DISCUSSION TITLE
Is there a way (maybe like launch parameters) to disable box colliders messages? and if so, is there a list of all posibles launch parameters?

### DISCUSSION THREAD

**Discussion by ItsRodrigoAl** [SmartlyDressedGames/Unturned-3.x-Community#4173] [Q&A]
Since the latest game updated this message has become more and more common, and even if it is not harmful, damn is annoying and now even more, it literally appears every 1 minute or less with only 20 players, and sometimes it shows only this message 1 time or then it is spammed and showed like 5 times
```
BoxColliders does not support negative scale or size.
The effective box size has been forced positive and is likely to give unexpected collision geometry.
If you absolutely need to use negative scaling you can use the convex MeshCollider. Scene hierarchy path "Player [Player]/Third/Skeleton/Spine/Right_Shoulder/Right_Arm/Right_Hand/Right_Hook/1488"
```

**Comment by Jdance-Media** [comment]
To add onto this, perhaps these specific errors could be assigned actual codes? That way there can be a dictionary on the docs website that goes into further detail about each error. Alongside that, then specific errors that aren't problematic can be disabled.

**Comment by MoltonMontro** [comment]
>is there a list of all posibles launch parameters 

https://docs.smartlydressedgames.com/en/stable/about/command-line.html

**Comment by SDGNelson** [comment]
Unfortunately not. A few years ago Unity would contact everyone asking once per year what devs needed and I kept saying "an option to turn off the box colliders warning". Ideally yeah the obvious answer would be to stop having negative scaled box colliders, but I disagree with that because:
- It's common in every game engine for level designers to flip objects with -1 scale on an axis, and it's silly that box colliders would need to be avoided for this
- The forced-positive-size does work how we want it to
